packages/blitz/src/render.rs

```diff
@@ -788,6 +788,58 @@ impl ElementCx<'_> {
                                 }
                             }),
                         );
+
+                    if let Some(underline) = &style.underline {
+                        let mut draw_underline = |x: f32, y: f32, w: f32| {
+                            let thickness = 1.0 / 18.0;
+                            let offset = 1.0 / 9.0;
```

Comment:
The thickness and offset should be come from the `style` and fall back to a value from `run_metrics`  (`glyph_run.run().metrics()`) rather than being hardcoded. See https://github.com/linebender/xilem/blob/main/masonry/src/text_helpers.rs#L126. I would suggest using `Option::unwrap_or` rather than the `match` statement in the linked code.